requirements/environment.yml

```diff
@@ -10,6 +10,7 @@ dependencies:
   - click
   - cerberus
   - dask
+  - dvc
```

Comment:
Hi @wint3ria, you'd rather add the DVC dependency to `setup.cfg`: `environment.yml` is sourced from it and will be overwritten if you update the locked dependencies. See [the docs](https://eradiate.readthedocs.io/en/latest/rst/developer_guide/dependencies.html) for more information about dependencies in Eradiate.

Reply:
Oh, thank you